source/index.txt

```diff
@@ -13,10 +13,19 @@ Introduction
 ------------
 
 Welcome to the documentation site for the official {+provider-long+}.
-You can add the provider to your .NET application as an object-relational mapper (ORM)
-to work with data in MongoDB.
+
+You can add the {+provider-short+} to your .NET application as an object-relational mapper
+(ORM) to work with data in MongoDB. The provider allows you to access and manipulate data in
```

Comment:
Issue:
From the style guide on "[allow](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#a)":
>  Use allow only when discussing permission. Avoid using allow to imply that a program, feature, or product permits a user to do something. Use you can instead. Enables is also acceptable.


Suggestion:
```suggestion
(ORM) to work with data in MongoDB. You can use the provider to access and manipulate data in
```